pkg/monitoring/rules/recordingrules/cluster_operator.go

```diff
@@ -0,0 +1,18 @@
+package recordingrules
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	"github.com/machadovilaca/operator-observability/pkg/operatorrules"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+var clusterOperatorRecordingRules = []operatorrules.RecordingRule{
+	{
+		MetricsOpts: operatormetrics.MetricOpts{
```

Comment:
@avlitman I dont see a reason to separate the recording rules. Its only making it harder to get the full list of recording rules.

Reply:
+1 will fix